test/functional/apps/discover/group1/_discover_histogram.ts

```diff
@@ -43,6 +43,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
       await security.testUser.setRoles(['kibana_admin', 'long_window_logstash']);
       await kibanaServer.uiSettings.replace(defaultSettings);
+      await PageObjects.timePicker.setDefaultAbsoluteRangeViaUiSettings();
```

Comment:
Making the test a big faster, since setting a different default time range in AdvancedSettings is faster than having to do so in UI